### ISSUE TITLE
Should we fail a build if AI file is accidentally missing?

### ISSUE THREAD

**Issue by samreid** [phetsims/chipper#173]
We could introduce a build step what verifies that AI files exist for each PNG file, and fails if it is missing _or_ not explicitly annotated as to why it is not needed.

**Comment by samreid** [comment]
Some AI files contain the art for several graphics.  Perhaps one way to handle this would be to add at attribute in the license.json that indicates where the original artwork is.  Is this worth the effort (compared to a code-review task that has us manually check)?  That is a question for the team.

**Comment by pixelzoom** [comment]
Is this a new requirement, that (in most cases) there is going to be a .ai file for each image file?  If so, then designers need to get on board with that, or it's going to create additional work for developers.  For example, when the PhET brand was being revised, assets were delivered as phet-brand-assets.ai.  I would not appreciate the added task of having to (repeatedly) divide that into multiple .ai files as the artwork is revised.

**Comment by samreid** [comment]
Alternatively, we could add a required "vectorArt" attribute for license.json entries which points to the AI file with the artwork.  Multiple PNGs could point to the same AI file.

**Comment by pixelzoom** [comment]
Or perhaps there should be a license.json file in the assets/ directory? There's currently no information about where the assets came from.

**Comment by samreid** [comment]
I can't think of any AI files in assets/ that were not created by PhET.  Do you know of any?

**Comment by pixelzoom** [comment]
No. But in Javaland, license.txt identified the team member that created the media file, that info might be useful for assets?  I can also envision a scenario where we get an audio file (or image) from some public domain site, then change the format to meet PhET's needs.

**Comment by samreid** [comment]
Knowing who created an asset would be valuable in case we need to iterate on it with them.  Right now the only ways to get that information would be from git commit logs or related issues, neither of which seems very reliable.

**Comment by zepumph** [comment]
I believe we have a good pattern for this, and licenses. We don't require much over in the assets/ dir, and leave it up to developers. Closing
